### DISCUSSION TITLE
Minimap Support

### DISCUSSION THREAD

**Discussion by Mark7625** [runelite/runelite#15228] [Q&A] [answered]
### Why Do we need this

We need this feature for 117 Plugin this will Allow us to add some extra effects like
  - Tile Changes matching the colors of the plugin (Like the Winter theme and more to come)
  - Showing Fake Tiles we add for certain areas 
  - Not showing any tiles from areas we hide
This will also allow allow people to have Shaded Minimap plugin, that I will put on the plugin Hub as ive seen it requested a few times by myself included 
#### What needs to be added
 
* A method to replace  the Minimap method that draws tiles
* SceneTileModel will also need these added and exposed 
   * SwColor
   * SeColor
   * NwColor
   * SeColor
   * NeColor
   * isFlat
   * overlayRgb
   * underlayRgb
   * istextured
* Rasterizer
  * colorPalette

**Comment by SirFancyBacon** [comment]
I second this feature if possible; I am working on modernizing Edgeville to a more 2010 feel and it no longer matches the minimap:
![image](https://user-images.githubusercontent.com/11658143/178087481-7c801724-9e34-45c3-8685-b0c2c960dd0d.png)
If the above methods would work that would be a huge step for the quality of graphics for OSRS

**Comment by Ferrariic** [comment]
Seems like a very popular request that would help multiple plugins, I hope they're able to evaluate this.

**Comment by Nightfirecat** [comment, marked as answer]
Added in dc98e98617f737d8834c3071fbd5452924f575cc
